sdk/src/actor.rs

```diff
@@ -62,6 +62,12 @@ pub fn create_actor(actor_id: ActorID, code_cid: &Cid) -> SyscallResult<()> {
     unsafe { sys::actor::create_actor(actor_id, cid.as_ptr()) }
 }
 
+/// Installs or ensures an actor code CID is valid and loaded.
+pub fn install_actor(code_cid: &Cid) -> SyscallResult<()> {
+    let cid = code_cid.to_bytes();
+    unsafe { sys::actor::install_actor(cid.as_ptr()) }
+}
+
```

Comment:
GitHub does not render the diff cleanly after switching the base branch to `master`. The `install_actor` syscall is already present in master. The branch history with multiple merges from ancilliary branches is confusing GH. I will have to close this PR, create a new branch with a clean diff, and open a new PR. I will preserve authorship in the commit history.